### PR TITLE
Cleanup reference manual lmp section

### PR DESCRIPTION
**Pull request by kprosise**
`reference-manual/linux` pages were checked and edited for grammar, spelling, and style.

Pages which were a better fit for the user-guide were relocated under lmP Customization; an index page for that section was also added.

Cleanup and possible relocation of linux-updating was skipped to avoid merge conflicts.

Redirects added for moved pages.

QA steps: checked rendered html. Ran linter within editor. Ran linkcheck. No issues to report, however this is a relatively large commit.

This commit addresses FFTK 2791
This commit addresses FFTK 1887
This commit addresses FFTK 1741
This commit addresses FFTK 1681
This commit applies to FFTK 1687
This commit applies to FFTK 2730
This commit applies to FFTK 2732
This commit applies to FFTK 2731
This commit applies to FFTK 1702
This commit applies to FFTK 988
This commit applies to FFTK 2733

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

This is a pretty large commit. Following cleanup commits will be a touch more atomic. For those reviewing, feel free to only review changes on pages/subjects you are familiar with.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for bcbe4e68e7fc04159eaaf1a667b95c6a3b1713d1 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2499/docs/artifacts/html/index.html

File: source/user-guide/lmp-customization/linux-building.rst [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+On macOS and Windows, see :ref:`ref-linux-dev-container` on setting up a containerized Linux build environment.
+
+You will also require at least 50GB of storage for a complete LmP build.
+
+Setup the Build Environment
+---------------------------
+
+On Debian-based Linux distributions, including Ubuntu, run::
+
+   $ sudo apt-get install coreutils gawk wget git-core diffstat unzip \
```
Comment (kprosise) [inline source/user-guide/lmp-customization/linux-building.rst:35]:
```suggestion
   $ sudo apt install coreutils gawk wget git diffstat unzip \
```

**Comment by doanac** [comment]
Docs for c396726b02236a04bccee639438a0a88bf6e1db9 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2500/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-wic-installer.rst [hunk 2]
```diff
@@ -20,35 +18,30 @@ definition for your target machine to ``image-efi-installer.wks``::
...
-is set to ``ttyS0,115200``, which can be done manually in grub (by editing
-the boot arguments before booting the ``install`` target) or by removing
-``console=tty0`` from the image installer by changing
-``lmp-factory-custom.inc``::
+If running QEMU without graphics support, make sure that the default console is set to ``ttyS0,115200``.
+This can be done manually in GRUB by editing the boot arguments before booting the ``install`` target.
+It can also be done by removing ``console=tty0`` from the image installer by changing ``lmp-factory-custom.inc``::
 
   $ cat meta-subscriber-overrides/conf/machine/include/lmp-factory-custom.inc
   APPEND:remove:intel-corei7-64 = "console=tty0"
```
Comment (kprosise) [inline source/reference-manual/linux/linux-wic-installer.rst:37]:
```suggestion
    APPEND:remove:intel-corei7-64 = "console=tty0"
```

**Comment by doanac** [comment]
Docs for 1839524455678ca7c92ec579fe0a249ff558baa4 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2501/docs/artifacts/html/index.html

**Review by angolini — APPROVED** [review]
Only minor suggestions, otherwise it's a LGTM

File: source/reference-manual/linux/building-sdk.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 .. _ref-building-sdk:
 
-Building SDK
-============
+Building the SDK
```
Comment (kprosise) [inline source/reference-manual/linux/building-sdk.rst:3]:
```suggestion
Building The Yocto Project Standard SDK
```

File: source/reference-manual/linux/linux-dev-mode.rst [hunk 4]
```diff
@@ -3,11 +3,10 @@
...
 =================================
 
-LmP provides the variable ``DEV_MODE`` that enables a development mode
-defined by the factory source code.
+LmP provides the variable ``DEV_MODE`` that enables a development mode defined by the Factory source code.
 
-The variable is defined as ``lmp:params:DEV_MODE`` and can be configured by updating a Factory's
-:ref:`factory-config.yml <def-lmp>` in ``ci-scripts.git`` with::
+The variable is defined as ``lmp:params:DEV_MODE``,
+and can be configured by updating a :ref:`factory-config.yml <def-lmp>` in ``ci-scripts.git`` with::
```
Comment (kprosise) [inline source/reference-manual/linux/linux-dev-mode.rst:9]:
```suggestion
and can be configured by updating the :ref:`factory-config.yml <def-lmp>` in ``ci-scripts.git`` with::
```

File: source/reference-manual/linux/linux-oss-compliance.rst [hunk 5]
```diff
@@ -1,39 +1,33 @@
...
-code tarball by default.
+FoundriesFactory configures the LmP to provide a license manifest and source code tarball by default.
 
-The license manifest can be found at
+The license manifest for a given Target can be found at:
 
 ``https://app.foundries.io/factories/<factory>/targets/<version>/artifacts/<machine>/other/<image>-<machine>.license.manifest``
 
-All the package's source code under GPLv2 or GPLv3 license family can be found at
+All the image's source code under the GPLv2 or GPLv3 license family can be found at:
```
Comment (kprosise) [inline source/reference-manual/linux/linux-oss-compliance.rst:30]:
```suggestion
All the image's packages source code under the GPLv2 or GPLv3 license family can be found at:
```

**Review by kprosise — COMMENTED** [review]
Suggestions from feedback

**Comment by doanac** [comment]
Docs for 8998943cede444331b98205d06eba13f6eaeb130 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2505/docs/artifacts/html/index.html

**Pull request merged**